jscomp/core/js_of_lam_float_record.ml

```diff
@@ -39,7 +39,7 @@ let get_double_feild (field_info : Lam_compat.field_dbg_info) e i =
 #end
   | Fld_record s 
   | Fld_module s 
-    -> E.array_index_by_int ~comment:s e i
+    -> E.dot e s
```

Comment:
Float record does not make too much sense in JS context, I will send a PR to remove such stuff